WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -117,11 +117,17 @@ struct AddEditCoupon: View {
                                                      editable: true,
                                                      fieldAlignment: .leading,
                                                      keyboardType: .decimalPad,
-                                                     inputFormatter: CouponAmountInputFormatter())
+                                                     contentColor: $viewModel.amountFieldColor,
+                                                     inputFormatter: CouponAmountInputFormatter()) { editingChanged in
```

Comment:
Nit: let's rename `editingChanged` to `beginningEditing` to clarify the intention to only validate when editing ends. How does that sound?

Reply:
Sure! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/7036/commits/0e2f384ff985f22133b9ae3f99af2725423a4f40